monitoring/volume_collector.go

```diff
@@ -0,0 +1,65 @@
+package monitoring
+
+import (
+	"github.com/sirupsen/logrus"
+
+	"github.com/prometheus/client_golang/prometheus"
+
+	"github.com/longhorn/longhorn-manager/datastore"
+)
+
+type VolumeCollector struct {
+	*baseCollector
+	volumeCapacityMetric   metricInfo
+	volumeActualSizeMetric metricInfo
+}
+
+func NewVolumeCollector(
+	logger logrus.FieldLogger,
+	nodeID string,
+	ds *datastore.DataStore) *VolumeCollector {
+
+	vc := &VolumeCollector{
+		baseCollector: newBaseCollector("volume", logger, nodeID, ds),
+	}
+
+	vc.volumeCapacityMetric = metricInfo{
+		Desc: prometheus.NewDesc(
+			prometheus.BuildFQName(longhornName, subsystemVolume, "capacity_bytes"),
+			"Configured size in bytes for this volume",
+			[]string{prometheusNodeIDLabel, prometheusVolumeIDLabel},
+			nil,
+		),
+		Type: prometheus.GaugeValue,
+	}
+
+	vc.volumeActualSizeMetric = metricInfo{
+		Desc: prometheus.NewDesc(
+			prometheus.BuildFQName(longhornName, subsystemVolume, "actual_size_bytes"),
```

Comment:
maybe just `used_bytes` ? 

This can be used to get the ratio of used / capacity and do alerting in the case when there are to many snapshots for a volume for example. (i.e. once ratio > 5 for example)

Reply:
I see elsewhere you use the term `usage` instead, so let's stick with that?

Reply:
Thank you! Fixed